modules/twitter.py

```diff
@@ -0,0 +1,27 @@
+from .base import Module
+from bs4 import BeautifulSoup
+import requests
+
+class Twitter(Module):
+    DESCRIPTION = 'Creates Twitter DM button'
+    def response(self, query, message):
```

Comment:
I would really prefer to use Twitter's official API down the line, but this is fine for now.